flashcard_runner.rb

```diff
@@ -0,0 +1,44 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+require './lib/card_generator'
+
+
+def start
+    cards = CardGenerator.new("cards.txt").cards
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+
+    puts "Welcome! You are playing trivia with 5 cards."
+    puts "---------------------------------------------"
+
+    card_number = 1
+    loop do 
+        card = round.current_card
+        puts "This is card number #{card_number} out of #{deck.count}."
+        puts card.question        
+        guess = gets.chomp.capitalize         
+        turn = round.take_turn(guess)                  
+        puts turn.feedback
+        card_number += 1
+    
+        if round.current_card == nil
+            break
+        end
+    end
+
+    puts "****** Game over! ******
+    You had #{round.number_correct} correct guesses
+    out of #{deck.count} for a total score of #{round.percent_correct}% correct."
+    puts "STEM - #{round.percent_correct_by_category(:STEM)}% correct"
+    puts "Geography - #{round.percent_correct_by_category(:Geography)}% correct"
+end
+    
+start
+
```

Comment:
You had asked for feedback around your runner file and the conventions around it. At this point in your developer career, I'm happy with what you were able to produce, and how you were able to get your game to run via this runner file. In the future, we'll challenge you to store as little code as possible outside of a class. Right now, you have code in this file that is not in a class. So, if you were to follow that convention, you could have a line of code in your runner file that was something like Game.start. Where Game is a new class that you created for the purpose of running the game. And start was a method that kicked off the flow of the game. The idea is that the flow of the game is stored in methods in that game class.